manifest.php

```diff
@@ -28,11 +28,11 @@
 	'label' => 'extension-tao-dac-simple',
 	'description' => 'extension that allows admin to give access to some resources to other people',
     'license' => 'GPL-2.0',
-    'version' => '5.0.1',
+    'version' => '5.1.0',
```

Comment:
```suggestion
    'version' => '6.0.0',
```

Reply:
From all discussions there was a summary. If requered extension changes major version then we change major version too. If not then we can change minor version.
In this extension generis version changed from 12.3 to 12.5